Examples/antsApplyTransforms.cxx

```diff
@@ -849,6 +880,30 @@ static void antsApplyTransformsInitializeCommandLineOptions( itk::ants::CommandL
   parser->AddOption( option );
   }
 
+  {
+  std::string description =
+    std::string( "Output image data type. " )
+    + std::string( "This is a direct typecast; output values are not rescaled. " )
+    + std::string( "Default is to use the internal data type (float or double). " )
+    + std::string( "uchar is unsigned char; others are signed. " )
+    + std::string( "WARNING: Outputs will be incorrect if values exceed the " )
```

Comment:
I think perhaps this should be more explicit, how is it incorrect? Clipped or rescaled?

Reply:
> I think perhaps this should be more explicit, how is it incorrect? Clipped or rescaled?

I changed the text to ..."incorrect (overflowed/reinterpreted)"... 
How's that?

It's not clipped or rescaled. It just goes through a raw typecast. I believe the behavior of particular some typecasts is defined and others are undefined or implementation-dependent, so there is not a short phrase to fully describe this. "overflowed/reinterpreted" was the best I could come up with.